charts/zac/values.yaml

```
@@ -89,7 +89,9 @@ brpApi:
     # -- If specified, enables the BRP Protocollering
```

this is not ok. `# -- ` style comments should only be used on leaf nodes, not at this level.
```suggestion
    # If specified, enables the BRP Protocollering
```

Also I think this comment is in the wrong place? The `originOin` does not enable BRP protocollering?